Services/COPage/classes/class.ilPageEditorGUI.php

```diff
@@ -31,6 +31,7 @@
  * @ilCtrl_Calls ilPageEditorGUI: ilPCBlogGUI, ilPCQuestionOverviewGUI, ilPCSkillsGUI
  * @ilCtrl_Calls ilPageEditorGUI: ilPCConsultationHoursGUI, ilPCMyCoursesGUI, ilPCAMDPageListGUI
  * @ilCtrl_Calls ilPageEditorGUI: ilPCGridGUI, ilPCGridCellGUI, ilPageEditorServerAdapterGUI
+ * @ilCtrl_Calls ilPageEditorGUI: ilPCCurriculumGUI, ilPCLauncherGUI
```

Comment:
It should be possible to move this to the ilPCCurriculumGUI, ilPCLauncherGUI classes and use a "isCalledBy" comment.